src/packages/mikroorm/src/plugins/connect-to-database.ts

```diff
@@ -1,18 +1,23 @@
 import { ApolloServerPlugin } from '@apollo/server';
 import { ConnectionManager, ConnectionOptions } from '../database';
 
+const connectionIds = new Set<string>();
+
 export const connectToDatabase = (
 	options: ConnectionOptions[] | ConnectionOptions
 ): ApolloServerPlugin => {
 	return {
 		serverWillStart: async () => {
 			if (Array.isArray(options)) {
 				for (const option of options) {
-					if (option.connectionManagerId)
+					if (option.connectionManagerId && !connectionIds.has(option.connectionManagerId)) {
+						connectionIds.add(option.connectionManagerId);
```

Comment:
Was connection manager not adding the ids before??

Reply:
no its not 😢